src/pyscaffold/templates/__init__.py

```diff
@@ -135,10 +135,20 @@ def setup_cfg(opts: ScaffoldOpts) -> str:
     Returns:
         str: file content as string
     """
+
     template = get_template("setup_cfg")
-    cfg_str = template.substitute(opts)
+
+    # template assumes oneliner `description`, this workaround avoids parsing errors:
```

Comment:
Could we write maybe?
# template needs single-line `description`, thus we pass only the first line of a multi-line description...